test/e2e/post-911-gib-status-helpers.js

```diff
@@ -0,0 +1,89 @@
+const mock = require('./mock-helpers');
+
+const enrollmentData = {
```

Comment:
This isn't critical, but since we're just testing for one thing to be rendered (since unit tests are testing the details), we can probably simplify this object to just a couple of fields.